.github/workflows/benchmarks.yml

```
@@ -11,13 +11,8 @@ on:
     branches: [ main ]
     paths:
       - 'pygmt/**/*.py'
-      - '!pygmt/tests/**'
-      - '.github/workflows/benchmarks.yml'
-  pull_request:
-    paths:
-      - 'pygmt/**/*.py'
-      - '!pygmt/tests/**'
       - '.github/workflows/benchmarks.yml'
+  # pull_request:
```

Add our standard hint about uncommenting the pull_request line
```suggestion
  # Uncomment the 'pull_request' line below to trigger the workflow in PR
  # pull_request:
```